src/app/helpers/spies.ts

```diff
@@ -1,21 +1,31 @@
 export const resetSpies = (spies: any[]) => {
-    spies.forEach(spyObj => {
-      Object.keys(spyObj).forEach(prop => {
-        if (typeof spyObj[prop] === 'function') {
-          spyObj[prop].calls.reset();
-        }
-      });
+  spies.forEach(spyObj => {
+    Object.keys(spyObj).forEach(prop => {
+      if (typeof spyObj[prop] === 'function') {
```

Comment:
there is no else for this statement which beats the logic of it being a conditional statement, so i guess you can as well remove the if and just point to what is returned by the if

Reply:
But I only want to reset spy functions ... if the object is not a function we do nothing to it ... If I remove the `statement` then some unwanted bugs might be introduced ... regardless, this statement wasn't authored by me, it just says so because the linter made some changes, but if you are keen you'll notice that it was introduced by a previous PR

Reply:
ooh...ok..no big deal as it does not break anything